### DISCUSSION TITLE
Is it possible to exclude files from sync?

### DISCUSSION THREAD

**Discussion by AleXSR700** [Natizyskunk/vscode-sftp#446] [Q&A]
Hi everyone,
I would like to exclude a database file from syncing.

I am using this extension to backup all the scripts etc. I am running on a server and I would like to backup everything except the database that is stored in that same directory. There is no need to back it up and given its size (>10 GB) it does not make sense to download it everytime.

Is this possible somehow?

Thank you
Alex

**Comment by jason-ivey** [comment]
Yes. 

```json
{
    "name": "iub",
    "host": "111121212",
    "protocol": "sftp",
    "port": 1212,
    "username": "121212",
    "password": "12121212",
    "remotePath": "/home/,
    "uploadOnSave": true,
    "useTempFile": false,
    "openSsh": true,
    "ignore": [
        ".vscode",
        ".git",
        ".DS_Store",
        "*__pycache__*",
        "venv",
        "venv_tool_366",
        "*.pyc",
        "sftpc.json",
        "sftp.json",
        "venv_366",
        "*sftpc.json"
    ]
}
```
